src/main/resources/modules/stroke_exm_105_r4.json

```diff
@@ -268,14 +236,14 @@
           }
         },
         {
-          "transition": "Terminal"
+          "transition": "Set_IPP"
```

Comment:
For 105, we have `NonElectiveEncounter.relevantPeriod ends during "Measurement Period"`, so we also need to make sure that the `Hospital_stay_end` is during the measurement period.